frontstage/templates/sign-in/sign-in.account-not-verified.html

```diff
@@ -1,12 +1,14 @@
 {% import 'partials/section.html' as section %}
 {% extends "layouts/_twocol.html" %}
 
-{% block page_title %}Trouble signing in? - ONS Business Surveys{% endblock %}
+{% block page_title %}Verify your email - ONS Business Surveys{% endblock %}
 
 {% block main %}
-    <h1 class="saturn">Trouble signing in?</h1>
+    <h1 class="saturn">Verify your email</h1>
 
-    <p>Please follow the link in the verification email we sent you.</p>
-    <p>If you haven't received an email please call us on 0300 1234 931 or email <a href="mailto:surveys@ons.gov.uk">surveys@ons.gov.uk</a></p>
+    <p>When you created your account we sent you an email to verify your email address.</p>
```

Comment:
This matches content, but i think it doesn't make sense. Shouldn't it be:

```
When you created your account we sent you an email to verify your account.
```

Reply:
Either makes sense in my mind, @shahi645. And both forms can been found elsewhere 👍

Reply:
Yeah i noticed that, as long as we are consistent i suppose